seed/static/seed/js/controllers/inventory_list_beta_controller.js

```diff
@@ -1222,8 +1282,9 @@ angular.module('BE.seed.controller.inventory_list_beta', [])
               const subFilters = _.map(_.split(filter.term, ','), _.trim);
               for (const subFilter of subFilters) {
                 if (subFilter) {
-                  const {operator, value} = parseFilter(subFilter)
-                  $scope.column_filters.push({column_name, operator, value})
+                  const {string, operator, value} = parseFilter(subFilter)
+                  const display = [column_name, string, value].join(' ')
```

Comment:
Do you think we could display the column's "display name" instead of the internal column_name here? Might be a better experience for users (e.g. showing `"Site EUI" < 10` instead of `site_eui < 10`)

Reply:
I had to collect all the display names on page load to support this but it's working.  It isn't too terribly helfpul for extra_data though!